testsuite/features/support/constants.rb

```diff
@@ -967,6 +982,12 @@
         sle-manager-tools-for-micro5-pool-x86_64-5.5
         sle-manager-tools-for-micro5-updates-x86_64-5.5
       ],
+    'sl-micro-6.0' =>
+      %w[
+        sl-manager-tools-for-micro6-pool-x86_64-6.0
+        sl-manager-tools-for-micro6-updates-x86_64-6.0
```

Comment:
The updates channel is not available as of now according to SCC, so this value still needs to be commented out.

Reply:
oh man thanks, I thought both were ready. Thanks for checking SCC

Reply:
now commented